### PR TITLE
Updated Docs

### PR DESCRIPTION
**Pull request by yeqbfgxjiq**
Added content to:
- `projects/welcome.md`
- `projects/redemptions.md`

The apps pages were a few commits behind MVP so I updated them with more content/links (and removed the emojis)
- `docs/apps/ all pages`

### CODE REVIEW

**Comment by lkngtn** [comment]
- I think in projects/welcome we should have the order of the projects match the order in the sidebar. 

(side note, but do you know if there is an issue with id collisions, can we just have the id for that page be welcome even though there is another welcome page with that id in a different section?)

- Perhaps we should use a table rather than a list for these projects since each entry has multiple pieces of data (name/link, blurb, status)?

- The blurb about 1hive doesn't really make it clear to me that it is references the website/blog/project as a whole. Sort of confusing imo. 

- No comments on anything else. Overall solid improvements! 👍

File: docs/projects/redemptions.md [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
 ---
 
-This needs to be written.
+An Aragon app that enables redeeming organizations tokens for organization assets.
+
+<br>
+
+## What is the Redemptions app?
+
+The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn within the organization for the underlying asset of the vault. It also allows anyone to purchase HONEY by depositing the underlying asset (ETH/DAI) in the vault.
```
Comment (kanaagawa) [inline docs/projects/redemptions.md:13]:
```suggestion
The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn within the organization for the underlying asset of the vault. 
```
As discussed in the ideas repo this is going to be the responsability of a new app: `Token Request`

File: docs/projects/redemptions.md [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+
+<br>
+
+## What is it good for?
+
+The redpemtions app lets you exchange your organization's native tokens for other tokens such as ETH DAI. This means that anyone can purchase your oganization's token, then use that token to create bounties to open issues that they want to see worked on. For example, 1Hive specializes in resources and mechanisms that support open source commons. Members (BEEs) in the 1Hive organization have skills that are relevant to this. If your project wants to better understand how bonding curves could help you bootstrap a new project, you could purchase HONEY from the redemptions vault, create an open Issue stating your problem and what you think a possible solution might look like, and then BEEs from 1Hive would work on that for you! BEEs who get the HONEY attached to bounties could then exchange it against the redemption vault for ETH or DAI to pay rent or go on vacation. This is great because you get cutting edge research/applications, 1Hive gets to sustainably work on things that further our mission, and the code/results of our work are available for anyone to learn from and utilize. Positive-sum sustainable open source commons :)
```
Comment (kanaagawa) [inline docs/projects/redemptions.md:19]: Maybe here as well state that people can't purchase Honey from the `Redemptions` app.

**Review by 0xGabi — APPROVED** [review]
Other than that looks great

File: docs/projects/redemptions.md (hunk 2 above)
Comment (yeqbfgxjiq) [reply docs/projects/redemptions.md:19]: Yeah my bad. I was under the impression that the redemptions app was a 2-way vault, but it's actually not. I'll update this section accordingly.

File: docs/projects/redemptions.md (hunk 1 above)
Comment (yeqbfgxjiq) [reply docs/projects/redemptions.md:13]: Yeah sorry. I got confused on the functionality. Your edit here fixes that though :)

**Comment by yeqbfgxjiq** [comment]
@lkngtn 

> I think in projects/welcome we should have the order of the projects match the order in the sidebar.

Agreed. I just copied them from the old 1Hive site as intros for the Projects section, but forgot to double check the sidebar

> do you know if there is an issue with id collisions, can we just have the id for that page be welcome even though there is another welcome page with that id in a different section?

No collisions as long as the welcome.md pages are in separate folders and indexed as such in sidebar.json

> Perhaps we should use a table rather than a list for these projects since each entry has multiple pieces of data (name/link, blurb, status)?

Open to it, but in my experience tables get clunky fast. A list is simpler, esp in the early stages. If it starts to feel hard to manage as we move towards V2 it might makes sense to do something like this. At the moment I'm not seeing the immediate benefit though.

 > The blurb about 1hive doesn't really make it clear to me that it is references the website/blog/project as a whole. Sort of confusing imo.

Yeah it was on the original website so I copied it over. I think it's important to highlight that 1Hive is dogfooding everything we're shipping and is leading by example. As such it's a "project" too, but the way it's presented is a bit confusing. I'll try to clean it up a bit, but let me know if you have more specific suggestions on how to best present that

**Pull request merged**